content/en/docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -90,11 +103,18 @@ algorithmSource:
 ...
 
 disablePreemption: true
-
 ```
 
-Although preemption of the scheduler is enabled by default, it is disabled if `PodPriority`
-feature is disabled.
+### Start an older version of Rescheduler in the cluster
+
+When priority or preemption is disabled, we must start Rescheduler v0.3.1
+(instead of v0.4.0) to ensure that critical Pods are scheduled when nodes or
```

Comment:
s/or/of.

Reply:
I actually meant "or". A cluster may not be under resource pressure and only one or more individual nodes are under resource pressure. In this case, Rescheduler helps scheduling those critical pods that must run on all nodes to be scheduled on the nodes under resource pressure.
Some times the whole cluster (and all of the nodes) may be under resource pressure. Rescheduler helps in this scenario as well.

Reply:
Makes sense. Thanks